### PR TITLE
Run Jenkins as nobody

### PR DESCRIPTION
**Pull request by colin-msphere**
Run Jenkins as `nobody` in UCR.

It is not possible to change the permissions or content of the original `JENKINS_HOME` volume. This creates a new volume with the proper owner and remaps some env vars to the new location.

* change `nobody` uid to `99` and add entry for `65534` uid
* remap `COPY_REFERENCE_FILE_LOG` and `JENKINS_HOME` env vars
* move `CMD` to a new script
* move nginx files to new location
* chmod dirs and files so either `nobody` uid can access them

### CODE REVIEW

**Comment by colin-msphere** [comment]
@mesosphere-ci ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

File: Dockerfile [hunk 1]
```diff
@@ -169,6 +169,8 @@ ADD https://infinity-artifacts.s3.amazonaws.com/statsd-jenkins/metrics-graphite.
 # disable first-run wizard
 RUN echo 2.0 > /usr/share/jenkins/ref/jenkins.install.UpgradeWizard.state
 
+ENTRYPOINT []
```
Comment (benclarkwood) [inline Dockerfile]: https://github.com/moby/moby/issues/3465#issuecomment-47598629 (for posterity)

File: universe/marathon.json.mustache [hunk 2]
```diff
@@ -3,7 +3,6 @@
   "cpus": {{service.cpus}},
   "mem": {{service.mem}},
   "instances": 1,
-  "user": "root",
```
Comment (benclarkwood) [inline universe/marathon.json.mustache]: We should just have the default be a user of `nobody`.

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
@mesosphere-ci ok to rebuild

**Comment by colin-msphere** [comment]
@mesosphere-ci ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok co rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

**Comment by colin-msphere** [comment]
ok to rebuild

File: Dockerfile [hunk 3]
```diff
@@ -166,27 +174,25 @@ ADD https://infinity-artifacts.s3.amazonaws.com/mesos-jenkins/mesos.hpi-${MESOS_
 ADD https://infinity-artifacts.s3.amazonaws.com/prometheus-jenkins/prometheus.hpi-${PROMETHEUS_PLUG_HASH} "${JENKINS_STAGING}/plugins/prometheus.hpi"
 ADD https://infinity-artifacts.s3.amazonaws.com/statsd-jenkins/metrics-graphite.hpi-${STATSD_PLUG_HASH} "${JENKINS_STAGING}/plugins/metrics-graphite.hpi"
 
+# change the config for $user
+# alias uid to $uid - should match nobody for host
```
[the remark this answers is not in the document]
Comment (colin-msphere) [reply Dockerfile]: No. I need to find a solution to the multiple UID issue.

**Pull request merged**